packages/utils/src/lib/reports/generate-md-report-categoy-section.ts

```diff
@@ -23,10 +23,15 @@ export function categoriesOverviewSection(
       { heading: '⭐ Score', alignment: 'center' },
       { heading: '🛡 Audits', alignment: 'center' },
     ],
-    categories.map(({ title, refs, score }) => [
+    categories.map(({ title, refs, score, isBinary }) => [
+      // @TODO refactor `isBinary: boolean` to `targetScore: number` #713
       // The heading "ID" is inferred from the heading text in Markdown.
       md.link(`#${slugify(title)}`, title),
-      md`${scoreMarker(score)} ${md.bold(formatReportScore(score))}`,
+      md`${scoreMarker(score)} ${md.bold(
+        formatReportScore(score),
+      )}${targetScoreIcon(score, isBinary === true ? 1 : undefined, {
+        prefix: ' ',
+      })}`,
```

Comment:
These calls could be made more compact by introducing a simple wrapper function:

```ts
export function binaryIconSuffix(score: number, isBinary: boolean | undefined): string {
  return targetScoreIcon(score, isBinary ? 1 : undefined, { prefix: ' ' });
}
```

```suggestion
      )}${binaryIconSuffix(score, isBinary)}`,
```

Also applies to lines 54-57 below.

Reply:
implemented the function and tests.